src/MoBi.Presentation/Presenter/SimulationChartPresenter.cs

```diff
@@ -40,9 +37,7 @@ protected override void MarkChartOwnerAsChanged()
 
       protected override string CurveNameDefinition(DataColumn column)
       {
-         var simulationForDataColumn = _simulations[column.Repository];
-
-         return _quantityDisplayPathMapper.DisplayPathAsStringFor(simulationForDataColumn, column);
+         return _curveNamer.CurveNameForColumn(_simulations[column.Repository], column, addSimulationName: false);
```

Comment:
The code change.

There's a namer in ComparisonChartsPresenter too that uses a different override of the mapper. Should we consolidate that into the CurveNamer as well?